modules/auxiliary/scanner/sap/sap_mgmt_con_brute_login.rb

```diff
@@ -167,7 +167,7 @@ def enum_user(user, pass, uri)
 
       report_cred(
         ip: rhost,
-        port: port,
+        port: datastore['RPORT'],
```

Comment:
```suggestion
        port: rport,
```
I think this is what @h00die was going for. Remote exploits have some nice convenience methods methods for accessing normalized datastore values including `rport` and `rhost` used above.